instrumentation/mybatis/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+dependencies {
+  compileOnly("com.google.auto.value:auto-value-annotations")
+  annotationProcessor("com.google.auto.value:auto-value")
+  library("org.mybatis:mybatis:3.2.0")
+  testImplementation("org.mockito:mockito-core")
+  testImplementation("org.mockito:mockito-junit-jupiter")
+  testImplementation("com.h2database:h2:1.4.191")
+}
+
+muzzle {
```

Comment:
usually muzzle block is right after the plugins block

Reply:
> usually muzzle block is right after the plugins block

I adjusted it.